filebeat/module/postgresql/log/test/postgresql-ubuntu-9.5.log-expected.json

```diff
@@ -0,0 +1,1092 @@
+[
```

Comment:
No need to add this expected file, some longer test files are only to check that the content can be parsed, not to check the generated events.